public/app/features/explore/Logs/Logs.tsx

```diff
@@ -183,6 +192,7 @@ class UnthemedLogs extends PureComponent<Props, State> {
           ...state.panelsState.logs,
           columns: logsPanelState.columns ?? this.props.panelState?.logs?.columns,
           visualisationType: logsPanelState.visualisationType ?? this.state.visualisationType,
+          labelName: logsPanelState.labelName,
```

Comment:
Could we refactor `labelName` to `labelFieldName` please? I think that name makes it a lot clearer what this prop is about.